samples/chatbot/src/main/java/io/quarkiverse/langchain4j/sample/chatbot/AugmentorExample.java

```diff
@@ -0,0 +1,32 @@
+package io.quarkiverse.langchain4j.sample.chatbot;
+
+import java.util.function.Supplier;
+
+import jakarta.enterprise.context.ApplicationScoped;
+
+import dev.langchain4j.model.embedding.EmbeddingModel;
+import dev.langchain4j.rag.DefaultRetrievalAugmentor;
+import dev.langchain4j.rag.RetrievalAugmentor;
+import dev.langchain4j.rag.content.retriever.EmbeddingStoreContentRetriever;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+
+@ApplicationScoped
+public class AugmentorExample implements Supplier<RetrievalAugmentor> {
+
+    private final EmbeddingStoreContentRetriever retriever;
+
+    AugmentorExample(EmbeddingStore store, EmbeddingModel model) {
+        retriever = EmbeddingStoreContentRetriever.builder()
+                .embeddingModel(model)
+                .embeddingStore(store)
+                .maxResults(20)
+                .build();
+    }
+
+    @Override
+    public RetrievalAugmentor get() {
+        return DefaultRetrievalAugmentor.builder()
```

Comment:
This was probably already proposed, but would it make sense to allow user to define just a `ContentRetriever` bean and it will be automatically wrapped into `DefaultRetrievalAugmentor` if there are no `RetrievalAugmentor` beans anywhere? Not sure it fits the Quarkus style?

Reply:
Yes this was discussed with @karesti  in the comments in this PR, she's going to send a PR for that

Reply:
I don't know if I'm understanding well enough, I'll pr a draft to check